avc_codec_registration.src.html

```diff
@@ -0,0 +1,168 @@
+!Participate: <a href="https://github.com/wicg/web-codecs">Git Repository.</a>
+!Participate: <a href="https://github.com/wicg/web-codecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/wicg/web-codecs/commits">https://github.com/wicg/web-codecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: web-codecs; for: WebCodecs; urlPrefix: https://wicg.github.io/web-codecs/#
+    type: attribute
+        text: VideoDecoderConfig.description; url: dom-videodecoderconfig-description
+    type: interface
+        text: EncodedVideoChunk; url: encodedvideochunk
+        text: VideoEncoder; url: videoencoder
+    type: dictionary
+        text: VideoEncoderConfig; url: video-encoder-config
+        text: VideoDecoderConfig; url: dictdef-videodecoderconfig
+</pre>
+
+<pre class='biblio'>
+{
+  "WebCodecs-Codec-Registry": {
+      "href": "https://wicg.github.io/web-codecs/codec_registry.html",
+      "title": "WebCodecs Codec Registry",
+      "publisher": "WICG"
+  },
+  "ITU-T-REC-H.264": {
+    "href": "https://www.itu.int/rec/T-REC-H.264",
+    "title": "H.264 : Advanced video coding for generic audiovisual services",
+    "publisher": "ITU",
+    "date": "June 2019"
+  }
+}
+</pre>
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+The codec string must begin with the prefix "avc1.", and contain a suffix of 6
+characters as described in Section 3.4 of [[rfc6381]].
+
+NOTE: The common "avc3." prefix is intentionally not included. This prefix does
+  not signal a material difference to decoder inputs, so it is omitted for
+  simplification. For the purposes of this registration, authors should simply
+  map "avc3." to "avc1.".
+
+VideoDecoderConfig description {#videodecoderconfig-description}
+================================================================
+
+A {{WebCodecs/VideoDecoderConfig.description}} may or may not be required by the
+AVC codec depending on the bitstream format of the AVC content to be decoded.
+
+If the {{WebCodecs/VideoDecoderConfig.description}} includes an
+`AVCDecoderConfigurationRecord`, as defined by [[iso14496-15]], section 5.3.3.1,
+it will be assumed that the bitstream is in "avc" format.
+
+NOTE: This format is commonly used in .MP4 files, where the player generally has
+    random access to the media data.
+
+If the {{WebCodecs/VideoDecoderConfig.description}} is not provided, it will be
+assumed that the bitstream is in “annexb” format.
+
+NOTE: "annexb" format is described in greater detail by [[ITU-T-REC-H.264]],
+    Annex B. This format is commonly used in live-streaming applications, where
+    including the SPS and PPS data periodically allows users to easily start
+    from the middle of the stream.
+
+VideoEncoderConfig extensions {#videoencoderconfig-extensions}
+==============================================================
+
+<pre class='idl'>
+<xmp>
+/* partial */ dictionary VideoEncoderConfig {
```

Comment:
@tabatkins, the partial above is in a comment because I couldn't figure out how make bikeshed aware that this extends a definition from another file in this repository (index.src.html). My understanding from reading the documentation is that all idl definitions are automatically exported such that this should just work?